azure-mgmt-appservice/src/main/java/com/microsoft/azure/management/appservice/implementation/AppServiceBaseImpl.java

```diff
@@ -436,6 +439,73 @@ public FluentImplT withExistingAppServicePlan(AppServicePlan appServicePlan) {
         return withOperatingSystem(appServicePlanOperatingSystem(appServicePlan));
     }
 
+    public FluentImplT withPublicDockerHubImage(String imageAndTag) {
+        ensureLinuxPlan();
```

Comment:
As users could create docker app service with windows runtime in portal, i think we should not check the app service plan here.

Reply:
I see. Web app got it. But function app still limit to Linux so far. I will no longer check this for web app.